engine/src/resource/store.cpp

```diff
@@ -90,14 +90,32 @@ bool Store::Buy(Customer& seller, std::string product_name, double quantity)
 
 bool Store::InStock(std::string product_name) {
     for(Product &in_store_product : stock) {
-        if(in_store_product == product_name && in_store_product.quantity > 0.0) {
-            return true;
+        if(in_store_product == product_name) {
+            return (in_store_product.quantity > 0.0);
         }
     }
 
     return false;
 }
 
+double Store::GetStock(std::string product_name) {
+    for(Product &in_store_product : stock) {
+        if(in_store_product == product_name ) {
+            return true;
+        }
+    }
+
+    return -1;
```

Comment:
Shouldn't this be `0` or is there some logic about it being `-1` for when there's no stock? If so, we should document that here too.

Reply:
It's an easy way to return an error, as product quantity is a resource and therefore, cannot be less than 0.

Reply:
@royfalk thanks. We probably should make a standard for this across the code-base and ensure we get it in the documentation